src/Rules/Resources/HowToFix.resx

```diff
@@ -497,7 +497,7 @@ Where appropriate, use a standard localized landmark type:
     ·         Use "navigation" for an area containing links for page or site navigation.
     ·         Use "search" for an area of the page containing search functionality.
 If none of the standard landmark types is applicable, provide a string that concisely describes its content.</value>
-    <comment>Brief guidance on how to fix an accessibility problem. {Locked="LocalizedLandmarkType"}</comment>
```

Comment:
We have 4 places where we list "banner", "complementary", "contentinfo", "form", "main", "navigation", and "search". 2 of these list all 7 items at the same level of the list. 1 lists "main", "navigation", and "search" as sub-items of "form", and 1 lists "navigation" and "search" as sub-items of "main".

I suspect that the intent is for all 7 items to be listed at the same level